### PR TITLE
consolidate shell/package.json into root package.json

### PR DESCRIPTION
**Pull request by smalls**
- `npm test` now runs all tests, but does not start a server.
- `npm test-with-start` starts the server, then runs all tests. This is
  now invoked from Travis.

### CODE REVIEW

**Comment by smalls** [comment]
Part of #826.

File: README.md [hunk 1]
```diff
@@ -10,18 +10,178 @@ Note that you need a **recent** version of Node because we use new ES6 features.
 
 ```
 $ ./tools/install
+$ ./tools/sigh
 ```
 
-## Test
+## Starting Arcs
+
+After the full build (`npm install && tools/sigh`) run: (note that `npm start`
```
Comment (shaper) [inline README.md]: Perhaps reference `./tools/install` rather than `npm install` here?

File: README.md [hunk 2]
```diff
@@ -10,18 +10,178 @@ Note that you need a **recent** version of Node because we use new ES6 features.
...
+See [devtools](devtools/README.md).
+
+## Testing
+
+The simplest way to run tests is to let the targets do all the work. These
+commands will install all packages, run a build, start a background server,
+run all the tests, and kill the background server:
+
+```
+npm install
```
Comment (shaper) [inline README.md]: Same as above -- perhaps `./tools/install` instead here.

**Comment by smalls** [comment]
As part of consolidating into a single `package.json`, the build will get a lot simpler... although it won't be until we remove `bower` that it gets to a single command. Although, I wonder if I can extend `npm install` to also run `bower` for devtools?

That might let us replace `tools/install` with a single `npm install`.

**Comment by smalls** [comment]
Yay! The build is back to green.

I confirmed that I can see `lint` as part of the build too. Once I merge I'll make sure that artifacts get pushed to PolymerLabs/arcs-live.

**Pull request merged**